### PR TITLE
Insert NuGet Build 4.6.0-preview2-4745 into sdk

### PR DESCRIPTION
**Pull request by nugetlurker**
Insert NuGet Build 4.6.0-preview2-4745 into sdk master branch

### CODE REVIEW

**Comment by mishra14** [comment]
@livarcocc @johnbeisner Can someone from the cli/sdk team take a look at the build issue please?

**Comment by mishra14** [comment]
One of the failure reasons is `NU1608`, which should be a warning, seems its been upgraded to an error.

File: build/Versions.props [hunk 1]
```diff
@@ -31,7 +31,7 @@
     <MicrosoftExtensionsDependencyModelVersion>2.0.0</MicrosoftExtensionsDependencyModelVersion>
     <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-25908-02</NETStandardLibraryNETFrameworkVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
-    <NuGetBuildTasksPackVersion>4.5.0-preview2-4529</NuGetBuildTasksPackVersion>
+    <NuGetBuildTasksPackVersion>4.6.0-preview2-4745</NuGetBuildTasksPackVersion>
     <NuGetPackagingVersion>4.5.0-preview2-4529</NuGetPackagingVersion>
```
Comment (johnbeisner) [inline build/Versions.props]: @mishra14
Why is "NuGetPackagingVersion" at the older version? Is this an oversight in the updating process?

**Comment by johnbeisner** [comment]
@mishra14 
NuGet.Build.Tasks.Pack/4.6.0-preview2-4745 has an added dependencies

      "NuGet.Build.Tasks.Pack/4.6.0-preview2-4745": {
        "type": "package",
        "dependencies": {
          "Microsoft.Build.Framework": "15.1.1012",
          "Microsoft.Build.Tasks.Core": "15.1.1012",
          "Microsoft.Build.Utilities.Core": "15.1.1012",
          "NETStandard.Library": "1.6.0",
          "NuGet.Commands": "4.6.0-preview2-4745"
        },

File: build/Versions.props (hunk 1 above)
Comment (mishra14) [reply build/Versions.props]: @johnbeisner Why do you have 3 different versions for NuGet?

Comment (johnbeisner) [reply build/Versions.props]: @mishra14 
The question is for you to field: Why would your update-tool update one and not the others?

Comment (mishra14) [reply build/Versions.props]: @johnbeisner I do not understand the need for 3 different variables. We can fix our automated PR process to update all of those, but if all three have the same value, then why not just use 1?

**Comment by mishra14** [comment]
@johnbeisner I will take a look at the dependency conflict introduced by Pack.

File: build/Versions.props (hunk 1 above)
Comment (johnbeisner) [reply build/Versions.props]: @mishra14 
"I do not understand the need for 3 different variables" - orchestrated build dictates the granularity of the packages

What is your process updating? And I'll base the others on the one your process is updating.

**Comment by johnbeisner** [comment]
Replaced by: https://github.com/dotnet/cli/pull/8281

**Comment by mishra14** [comment]
@johnbeisner this will also need to go in once the CLI PR is in. I have updated the build number to match the one in the CLI PR.

**Comment by johnbeisner** [comment]
@mishra14 
You will open a new PR to insert 4.6.0-preview2-4759 - correct

**Comment by mishra14** [comment]
new PR: https://github.com/dotnet/sdk/pull/1831

**Pull request closed without merging**